src/mapml/handlers/ContextMenu.js

```diff
@@ -629,19 +638,22 @@ export var ContextMenu = L.Handler.extend({
       case 77: //M KEY
         this._copyMapML(e);
         break;
-      case 70: //F KEY
-        this._goForward(e);
-        break;
+      // case 70: //F KEY
```

Comment:
We shouldn't leave commented code here; I think you have replace F with Alt+Right-Arrow, so maybe this is redundant now?